src/gui/previewselect.h

```diff
@@ -54,20 +55,31 @@ class PreviewSelect: public QDialog, private Ui::preview
         NB_COLUMNS
     };
 
-    PreviewSelect(QWidget* parent, BitTorrent::TorrentHandle *const torrent);
+    PreviewSelect(QWidget *parent, BitTorrent::TorrentHandle *const torrent);
     ~PreviewSelect();
 
 signals:
     void readyToPreviewFile(QString) const;
 
+protected:
+    void showEvent(QShowEvent *) override;
+
 protected slots:
     void on_previewButton_clicked();
     void on_cancelButton_clicked();
 
 private:
+    void loadSettings();
+    void saveSettings();
+
     QStandardItemModel *m_previewListModel;
     PreviewListDelegate *m_listDelegate;
     BitTorrent::TorrentHandle *const m_torrent;
+    bool m_headerState = false;
```

Comment:
headerState isn't appropriate name for bool variable! What does it mean "header state is true" or "header state is false"? When I see in the code `if (!m_headerState)` , I don't know what was going on. Please use a more meaningful name.

Reply:
done, renamed to `m_headerStateRestored`